examples/animation/double_pendulum.py

```diff
@@ -51,7 +50,7 @@ def derivs(state, t):
     return dydx
 
 # create a time array from 0..t_stop sampled at 0.02 second steps
-dt = 0.02
+dt = 0.01
```

Comment:
Please check if this is really necessary. This doubles the number of frames and thus likely the size of the generated file.

Reply:
I decreased the end time by a factor of two as well; the shorter time step does appear to improve the accuracy.

Reply:
But that means that you're only going half the path from before. Please check if that's still a reasonable path length. I'm afraid that might be a bit short.

I can imagine that the shorter time step improves accuracy. But does that matter here? Alternatively, one could also render only every second step, but that adds additional compelity to the code and I'm not sure that's worth it.

Reply:
I did check, the path length is fine.